app/presenters/product_mailchimp_presenter.rb

```diff
@@ -14,7 +14,7 @@ def json
           id: Digest::MD5.hexdigest(product.id.to_s),
           title: product.name || "",
           description: product.description || "",
-          url: "#{ENV['BASE_URL']}/#{product.category&.permalink || 'products'}/#{product.slug}",
+          url: "#{Rails.application.routes.url_helpers.spree_url}/#{product.category&.permalink || 'products'}/#{product.slug}" || "",
```

Comment:
Let's consider using `Spree::Core::Engine.routes.url_helpers.product_path(product.slug)`.

Reply:
@piotrleniec-spark we need to return an absolute URL with the host, `_path` helpers return relative path